poom/main_menu.py

```diff
@@ -278,18 +272,46 @@ def __init__(self, context, zombie: Zombie) -> None:
         )
         self.fps = UIDropDownMenu(
             ["on", "off"],
-            "off",
-            pg.Rect((width - 200) // 2, height * 0.78, 150, 50),
+            "off" if not exchanger["fps_tick"] else "on",
+            pg.Rect((width - 200) // 2, height * 0.71 + 40, 150, 50),
             manager,
         )
 
     def on_click(self, event) -> None:
+        global screen, width, height, size, background, manager
         if event.type == pygame_gui.UI_HORIZONTAL_SLIDER_MOVED:
             self.current_volume.set_text(f"{int(self.volume.current_value * 100)} %")
+            exchanger["volume"] = int(self.volume.current_value * 100)
+            pg.mixer.music.set_volume(self.volume.current_value)
+            with open(root / "assets" / "settings.json", "w") as file:
+                json.dump(exchanger, file)
         if event.type == pygame_gui.UI_BUTTON_PRESSED:
             if event.ui_element == self.back:
                 manager.clear_and_reset()
                 self._context.scene = WelcomeScene(self._context, self.zombie)
+        if event.type == pygame_gui.UI_DROP_DOWN_MENU_CHANGED:
+            if event.ui_element == self.graphics:
+                exchanger["quality"] = self.graphics.selected_option
+            if event.ui_element == self.screen_size:
+                new_size, ratio = self.screen_size.selected_option.split()
+                width, height = map(int, new_size.split("x"))
```

Comment:
Screen size can be stored as
```json
{
  "screen_size": {
    "width": 800,
    "height": 600
  }
}
```

After that, it can be easy formatted to `<width>x<height>`